### PR TITLE
Only use inotify on linux

### PR DESCRIPTION
**Pull request by pjvandehaar**

### CODE REVIEW

File: gunicorn/reloader.py [hunk 1]
```diff
@@ -53,12 +53,14 @@ def run(self):
...
             time.sleep(self._interval)
 
-try:
-    from inotify.adapters import Inotify
-    import inotify.constants
-    has_inotify = True
-except ImportError:
-    has_inotify = False
+has_inotify = False
+if sys.platform == 'linux':
```
Comment (berkerpeksag) [inline gunicorn/reloader.py]: We still support Python 3.2 so

```py
if sys.platform.startswith('linux'):
    ...
```

would be better.

Comment (pjvandehaar) [inline gunicorn/reloader.py]: Good catch.

**Comment by tilgovi** [comment]
LGTM

**Comment by berkerpeksag** [comment]
Thanks!

**Pull request merged**